src/common/Screen.js

```diff
@@ -44,7 +51,14 @@ class Screen extends React.Component {
           barStyle="dark-content"
           hidden={this.props.orientation === 'LANDSCAPE'}
         />
-        <ModalNavBar title={title} popRoute={popRoute} nav={nav} />
+        <View>
+          <ModalNavBar
```

Comment:
As I mention below, I think you should create a new component called `SearchNavBar` and decide here whether you need to show `SearchNavBar` or `ModalNavBar`